docs: make breakdown the first step (#224)
infracost/docs: docs/getting_started.md

The key can be retrieved with `infracost configure get api_key`.

### 3. Show cost estimate breakdown
Infracost parses the project locally to determine resource types and quantities needed to calculate costs. The [`--path` flag](/docs/features/cli_commands/#breakdown) can point to a Terraform directory or plan JSON file.

```shell
# You can also clone https://github.com/infracost/example-terraform
cd my-terraform-project

# Terraform variables can be set using --terraform-var-file or --terraform-var
infracost breakdown --path . --terraform-parse-hcl
```

> Example output:
> <img src={useBaseUrl("img/screenshots/get-started-breakdown.png")} alt="Infracost breakdown command" />

Reviewer: The spacing looks weird between the image and the tip:
```suggestion
<p>
Example output:
<img src={useBaseUrl("img/screenshots/get-started-breakdown.png")} alt="Infracost breakdown command" />
</p>
```

:::tip
Infracost can also estimate [usage-based resources](/docs/features/usage_based_resources/) such as AWS S3 or Lambda
:::

### 4. Show cost estimate diff

> <ol type="a">

Reviewer: I don't like the a, b, c list.  I think I would prefer first, next, then paragraphs.

  <li>Generate Infracost JSON file as the baseline:</li>

  ```shell
  infracost breakdown --path . \
      --terraform-parse-hcl --format json --out-file infracost-base.json
  ```

>   <li>Edit something in your Terraform project, such as an instance type.</li>

Reviewer: I'm guess people will skip this step since there's no code block, so maybe something like:
````suggestion
  <li>Edit something in your Terraform project.  If you're using our sample project, open `main.tf` and change the instance type:</li>
  ```shell
  vim main.tf
  ```
````

  <li>Generate a diff by comparing the latest code change with the baseline:</li>

  ```shell
  infracost diff --path . \
      --terraform-parse-hcl --compare-to infracost-base.json
  ```
</ol>

> Example output:
> <img src={useBaseUrl("img/screenshots/get-started-diff.png")} alt="Infracost diff command" />

Reviewer: ```suggestion
<p>
Example output:
<img src={useBaseUrl("img/screenshots/get-started-diff.png")} alt="Infracost diff command" />
</p>
```
